### PR TITLE
[TwigBundle] remove service when base class is missing

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
What's the context? Doesn't the container already remove the service? How did you spot this?

**Comment by nicolas-grekas** [comment]
Oups, this triggers a deprecation :)

**Comment by xabbuh** [comment]
Here is an example: https://travis-ci.org/FriendsOfSymfony/FOSRestBundle/jobs/618583998#L305

File: src/Symfony/Bridge/Twig/TwigEngine.php [hunk 1]
```diff
@@ -11,8 +11,6 @@
 
 namespace Symfony\Bridge\Twig;
 
-@trigger_error('The '.TwigEngine::class.' class is deprecated since version 4.3 and will be removed in 5.0; use \Twig\Environment instead.', E_USER_DEPRECATED);
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Twig/TwigEngine.php]: what about using a check on another thing that has been removed in 5.0 - and keep this runtime deprecation?
eg `method_exists(AssetExtension::class, 'getName')` could work, looking at the diff between 5.0 and 4.4?

Comment (xabbuh) [reply src/Symfony/Bridge/Twig/TwigEngine.php]: make sense

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**